Foundation/URLSession/URLSession.swift

```diff
@@ -195,6 +195,7 @@ open class URLSession : NSObject {
     fileprivate static let registerProtocols: () = {
 	// TODO: We register all the native protocols here.
         let _ = URLProtocol.registerClass(_HTTPURLProtocol.self)
+	let _ = URLProtocol.registerClass(_FTPURLProtocol.self)
```

Comment:
Why is this indentation like that? Also, you don't need to write `let _ =`; it's just `_ =`.